src/index.js

```diff
@@ -17,6 +17,10 @@ let store = createStore(reducer,
   )
 );
 
+// store.subscribe(() => {
```

Comment:
Just so you know, there is a redux dev tools extension for chrome you can use to browse the store, I used it in the talk.